sdk/clientcore/annotation-processor/src/main/java/io/clientcore/annotation/processor/utils/RequestBodyHandler.java

```diff
@@ -199,7 +200,6 @@ public static void setContentTypeHeader(BlockStmt body, String contentType) {
      * @param parameterName The name of the parameter to be serialized.
      */
     public static void handleRequestBodySerialization(BlockStmt body, String parameterName) {
-        body.tryAddImportToParentCompilationUnit(SerializationFormat.class);
```

Comment:
Don''t we need this  import here as well?

Reply:
Added the import back here.